examples/demo/modules/blog/cms_blog_node_type.e

```diff
@@ -17,10 +17,11 @@ feature {NONE} -- Initialization
 	default_create
 		do
 			Precursor
-			create {ARRAYED_LIST [like available_formats.item]} available_formats.make (3)
+			create {ARRAYED_LIST [like available_formats.item]} available_formats.make (4)
 			available_formats.extend (create {PLAIN_TEXT_CONTENT_FORMAT})
 			available_formats.extend (create {FILTERED_HTML_CONTENT_FORMAT})
 			available_formats.extend (create {FULL_HTML_CONTENT_FORMAT})
+			available_formats.extend (create {CMS_EDITOR_CONTENT_FORMAT})
```

Comment:
I added a CMS_EDITOR_CONTENT_FORMAT .. since the other usually convert new line, to <br/> html tag.
and the result was not great, so with CMS_EDITOR_CONTENT_FORMAT .. this is really pure HTML accepted.